test/test_cmake.rb

```diff
@@ -87,4 +127,19 @@ def test_cmake_command_configuration
       assert_equal("asdf", MiniPortileCMake.new("test", "1.0.0", cmake_command: "xyzzy").cmake_cmd)
     end
   end
+
+  private
+
+  def write_cmake_help(generator_type)
+    cmake_script = Tempfile.new('cmake_help')
+    cmake_script.puts <<~SCRIPT
```

Comment:
This shell script only works on UNIX! I guess I have to write a Windows version.

Reply:
Shebangs aren't recognized on windows, but you can probably run the shell-script as is if the command is `sh` and the script is the argument.

See https://github.com/flavorjones/mini_portile/blob/main/lib/mini_portile2/mini_portile.rb#L135-L139 for inspiration!

Reply:
Yeah, that would work too. I just updated this pull request to omit the shebang.